src/main/tables/83_container_ports.sql

```diff
@@ -0,0 +1,21 @@
+SET search_path = public, pg_catalog;
+
+-- Contains the the port information for a tool's container. This is used for
+-- interactive apps.
+
+CREATE TABLE container_ports (
+  -- primary key
+  id uuid UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
```

Comment:
The primary keys and foreign keys in `container_ports` and `interapps_proxy_settings` don't appear to be defined as such at the moment. Are there some constraints files that haven't been checked in yet?

Reply:
I accidentally did that work in another fork. Just moved it over here.